assignments/arrays.js

```diff
@@ -74,31 +323,136 @@ let inventory = [
 // [12, 14]
 
 // ==== Challenge 1 ====
-// The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*`);
+// The dealer can't recall the information for a car with an id of 33 on his lot. 
+// Help the dealer find out which car has an id of 33 by logging the car's year, make, 
+// and model in the console log provided to you below:
 
-// ==== Challenge 2 ====
-// The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+const object33 = inventory[33];
+// console.log(inventory[33]);
+console.log(`Car 33 is a ${Object.values(object33)[3]} ${Object.values(object33)[1]}  ${Object.values(object33)[2]} `);
+
```

Comment:
The 33rd object is indexed in the array as 32. Remember, arrays start counting at 0 not 1.